.gitignore

```diff
@@ -1,3 +1,4 @@
 /dist/
 node_modules/
 npm-debug.log
+/.rpt2_cache
```

Comment:
This is to ignore the cache directory generated by rollup-plugin-typescript2.
https://github.com/ezolenko/rollup-plugin-typescript2#plugin-options